example/prepare_binaries.sh

```diff
@@ -0,0 +1,34 @@
+export NBS_ROOT=".."
```

Comment:
no need to `export NBS_ROOT` 
you only export when you want to pass it as environment to some other program you invoke

by the way instead of redefining all possible binaries you can
`export LD_LIBRARY_PATH=...` once in this file (all the .so are copied to multiple location so it's enough to use some location for example the NBSD_BIN directory)

After that you source this file and you can invoke all the binaries with LD_LIBRARY_PATH passed correctly to the process and without adding the `LD_LIBRARY_PATH=...` prefix before the command.
For the sudo invoked process you can use `sudo -E <cmd>` to preserve environment and pass it to process invoked with sudo